db_structure.sql

```diff
@@ -34,4 +35,10 @@ CREATE TABLE ProductReviews (
     review TEXT NOT NULL ,
     rating NUMERIC CHECK (rating between 0 and 5 ) NOT NULL,
     UNIQUE(user_id, product_id)
-);
\ No newline at end of file
+);
+
+CREATE TABLE Categories (
+    id INT PRIMARY KEY,
+    name VARCHAR(255),
+    slug VARCHAR(255)
+)
```

Comment:
Add a newline to the end of this file so that this indicator disapears from the diff view

![image](https://github.com/damdiana/beauty-app/assets/11327761/36d87e60-8d4e-453a-8b4a-a927131e88be)